com.unity.netcode.gameobjects/Runtime/Core/NetworkManager.cs

```diff
@@ -43,6 +43,8 @@ public class NetworkManager : MonoBehaviour, INetworkUpdateSystem
 #endif
 
         private const double k_TimeSyncFrequency = 1.0d; // sync every second, TODO will be removed once timesync is done via snapshots
+        private const float k_DefaultBufferSizeSec = 0.05f; // todo talk with UX/Product, find good default value for this
```

Comment:
FWIW, I think 50ms might be too small here.

Reply:
This will be the default minimum latency set for all games using Multiplayer. 50ms already seems pretty high to me